Python/Github_Traffic/README.md

```diff
@@ -0,0 +1,85 @@
+# Github_Traffic
+
+This script, can help one determine all the unique viewers on his/her Repository.
+By using `access-token` one can determine the viewers and number of time they viewed the Repo.
+Currently data of only 14 days is possible (limitation of GitHib-API), but in future by using the GraphQL queries, this limitation can be answered.
+Te script uses PickleDB, the data can be imported in CSV as well, although the functionality hasn't been added till this point of time.
+
+## Note
+
+This script can be converted in a Bot, Action or GitHub app!!!
+
+
+## Dependency
+
+- githubpy
+- PickleDB
```

Comment:
Any particular reasons to use PickleDB? I'm not a DB-person so would like to have a further clarity on this.

Reply:
Definitely, TBH I am planning to make an action or a full-fledged app for this, (Like the dynamic star graph you have in README). For that reason I need to upload results of all the repositories so that I can sort them by most viewed repo of mine (Just an example), Pickle is fast and small weight, perfect for this purpose.

Reply:
Awesome! Thanks